site/docs/v1/tech/apis/identity-providers/_identity-provider-tenant-configuration.adoc

```diff
@@ -0,0 +1,14 @@
+[field]#identityProvider.tenantConfiguration# [type]#[Map<UUID, Object>]# {optional_tag} [since]#Available since 1.32.0#::
+The configuration for each Tenant that the identity provider is enabled for.
+
+[field]#identityProvider.tenantConfiguration``[tenantId]``.data# [type]#[String]# {optional_tag} [since]#Available since 1.32.0#::
+An object that can hold any information about the identity provider tenant configuration that should be persisted.
```

Comment:
We aren't exposing this as custom data, this field is just internally used for the `@InternalJSONColumn` annotation.

Reply:
Ahh yes, thank you.